01-messaging.md

```diff
@@ -243,6 +252,8 @@ The receiving node:
     - MUST ignore the bit.
   - upon receiving unknown _even_ feature bits that are non-zero:
     - MUST fail the connection.
+  - upon receiving `networks` containing no common chains
```

Comment:
what is the prescribed action if some chains intersect but not all? would think receivers should restrict all messages they send to this intersection?

Reply:
Hmm, like @roasbeef I haven't thought too hard about multi-chain implementation. 

Hmm, I think that addition (SHOULD) would be positive, yes.